### DISCUSSION TITLE
(Editor) Option for flipping horizontally/vertically relative to grid rotation and position

### DISCUSSION THREAD

**Discussion by ShiiTsuin** [ppy/osu#28574] [Ideas]
Just watched the most recent lazer changelog video about the editor and saw that olibomby's implementation of grid scaling, position and rotation was added to the most recent version of lazer.

It's only a minor Quality of Life improvement, but it would be nice if the flip selection horizontally/vertically shortcuts could work along the more visible lines of the grid, rather than just along y=0 and x=0. It'd just make it easier to create patterns that use specific lines of symmetry.

(flip currently ignoring grid)
![OkVBLvS](https://github.com/ppy/osu/assets/30865484/9363c837-dff0-4127-a0b0-47e0469c4d25)

**Comment by peppy** [comment]
Seems valid.
